casual/casual-api/src/main/java/se/laz/casual/config/Inbound.java

```diff
@@ -32,6 +36,11 @@ public boolean isUseEpoll()
         return useEpoll;
     }
 
+    public long getInitialDelay()
+    {
+        return initialDelay;
+    }
+
     @Override
     public boolean equals( Object o )
     {
```

Comment:
Can't add a comment down at toString, but it should include the new `initialDelay`